### ISSUE TITLE
Missing change user role in 3.2.1 

### ISSUE THREAD

**Issue by mkeenan750** [moodlerooms/moodle-theme_snap#71]
Hi i have 3.2.1 version of moodle (only available via my host) and I realise that this current version isnt stable yet on 3.2, just wanted to let you know of a small bug I have seen in current version of snap, the change user menu, switch from admin student is not available, when you switch back to boost its there, thought I should report it

**Comment by gthomas2** [comment]
Thank you for reporting this bug. We are aware of this issue and I believe it has been fixed very recently. Please can you refresh your branch. BTW - the fix we employed was to put the switch role option into the course menu.

**Comment by mkeenan750** [comment]
Hi gthomas2,

thanks for your response, I have updated and all fixed.

I know that the stable release doesn't support yet, 3.2.3 but I was wondering if there was a fix coming related to video, I have embedded a youtube video and it appears and I can press play but the controls and video in all devices dosen't work correctly, in mobile or tablet it plays, desktop it does audio only, and in all devices there is a loading icon even when the clip is playing, (see screenshot, taken on desktop, audio is playing no visual moving) https://ibb.co/ecLtJF

I just have seen comments on web relating to snap and video, so I really want to make sure your theme is able to display video well.

Any feedback would be appreciated.
